server/models/schema.sql

```diff
@@ -1812,6 +1812,7 @@ CREATE TABLE `stock_movement` (
   `is_exit`         TINYINT(1) NOT NULL,
   `user_id`         SMALLINT(5) UNSIGNED NOT NULL,
   `reference`       INT(11) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `invoice_uuid`    BINARY(16) NULL,  
```

Comment:
I thought this is what the `document_uuid` was for?

Reply:
I don't think so, `document_uuid` was for the document generated for a stock movement (SM.X.Y).

Reply:
The document meaning the `voucher`?